src/test/resources/no/nav/tilleggsstonader/sak/vedtak/barnetilsyn/validering_stonadsperioder.feature

```diff
@@ -15,16 +15,3 @@ Egenskap: Beregning barnetilsyn - validering av stønadsperioder
     Når beregner
 
     Så forvent følgende feil: Kan ikke innvilge når det ikke finnes noen vedtaksperioder
-
-  Scenario: Finner ingen aktiviteter på behandling
```

Comment:
Har vi noe liknende for den nye koden?

Reply:
Ja!

https://github.com/navikt/tilleggsstonader-sak/blob/2511ccc728fc4905da01e8b3905919ddd08bc1dd/src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/barnetilsyn/beregning/TilsynBarnVedtaksperiodeValidingerService.kt#L102:L104